config/config.go

```diff
@@ -1086,6 +1097,62 @@ func (c *Config) CheckLoggerConfig() error {
 	return nil
 }
 
+// CheckNTPConfig() check for incorrectly configured NTP server config entires and load sane defaults
+func (c *Config) CheckNTPConfig() {
+	m.Lock()
+	defer m.Unlock()
+
+	if c.NTPClient.AllowedDifference == nil {
+		c.NTPClient.AllowedDifference = new(time.Duration)
+		*c.NTPClient.AllowedDifference = 50000000
```

Comment:
Please create a const default value and use it instead of the above hardcoded value